Jobs/src/Job.IdentityDeletion/Job.IdentityDeletion.csproj

```diff
@@ -1,9 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Worker">
 
     <PropertyGroup>
-        <UserSecretsId>dotnet-Devices.Jobs.IdentityDeletion-553687ad-d9d2-47b9-88c5-a8869b426c22</UserSecretsId>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
```

Comment:
These can and should be removed. We want to control this via our `Directory.Build.props` files.

Reply:
Changes made.